README.md

```diff
@@ -74,6 +74,23 @@ generateRoutes(controllerList, routes, true)  // true = console info endpoints
 app.use(routes)
 ```
 
+#### Generate Routes With Herbarium
+
+As an alternative in a simpler way, if you already use Herbarium, a centralized and standardized repository and discovery service for Herbs objects, you can automatically generate and use new express routes:
```

Comment:
`As an alternative in a simpler way....`

This is not an alternative way, this is going to be the main way to use this library. The other ways should be "Advanced" and the doc should reflect that